src/views/List.jsx

```diff
@@ -12,35 +16,38 @@ export function List({ data }) {
 		item.name.toLowerCase().includes(searchItem.toLowerCase()),
 	);
 
+	const listName = listPath.slice(listPath.indexOf('/') + 1);
```

Comment:
Nice! Instead of using `slice`, consider using `split` to make the code more straightforward and readable. For example, `listPath.split('/')[1]` better conveys that we're splitting on the `/` and taking the value after it! 🚀